src/stream_manager.h

```diff
@@ -243,6 +258,11 @@ struct CUstream_st {
 
   pthread_mutex_t m_lock;  // ensure only one host or gpu manipulates stream
                            // operation at one time
+
+  // SST related, use to record the stream is requested to synchronize
+  bool m_requested_synchronize = false;
+  // Whether this is the default stream
+  bool is_stream_zero = false;
```

Comment:
why use a bool for this? Why not just test if the m_uid is 0? Or is this id not the streamid?

Reply:
Technically yes, the `m_uid` will be `0` for the zero stream as it is first one being constructed, but it is a rather implicit property, so I made explicit calls to mark the zero stream.